packages/format-library/src/link/inline.js

```diff
@@ -149,9 +149,9 @@ class InlineLinkUI extends Component {
 
 		if ( isCollapsed( value ) && ! isActive ) {
 			const toInsert = applyFormat( create( { text: url } ), format, 0, url.length );
-			onChange( insert( value, toInsert ) );
+			onChange( insert( value, toInsert ), { focus: true } );
 		} else {
-			onChange( applyFormat( value, format ) );
+			onChange( applyFormat( value, format ), { focus: true } );
```

Comment:
This API is interesting is it possible to call onChange without any change? So for example in the color picker when the popovers close and we know that we will not apply additional color changes we would call onChange to move the focus back to the rich text as the inline link does.

Reply:
this is just what I was looking for, it works well. Thanks a lot for all your hard work @ellatrix @jorgefilipecosta